src/ui/public/utils/decode_geo_hash.js

```diff
@@ -46,6 +51,13 @@ export function decodeGeoHash(geohash) {
   return { latitude: lat, longitude: lon};
 }
 
+/**
+ * @method refine_interval
+ * @param interval {Array} [long, lat]
+ * @param cd {Number}
```

Comment:
__nit:__ Kind of useless JsDoc comments (except maybe for `[long, lat]` for `interval` parameter) :) And `@return` is wrong.

Reply:
yeah, I didn't want to trash the comments that were here already but you're right that this should be useful or removed.